### PR TITLE
added index and pages

### PR DESCRIPTION
**Pull request by Wowzies**
i just added a html boilerplate and created a pages folder

### CODE REVIEW

**Comment by nefultre** [comment]
Hi! did you validate the html using a web validator?

**Comment by Wowzies** [comment]
> Hi! did you validate the html using a web validator?

no thats weak

**Comment by nefultre** [comment]
> > Hi! did you validate the html using a web validator?
> 
> no thats weak

hey it looks like youre missing an alt text attribute on the img. can you add that before we merge?

**Comment by Wowzies** [comment]
> > > Hi! did you validate the html using a web validator?
> > 
> > 
> > no thats weak
> 
> hey it looks like youre missing an alt text attribute on the img. can you add that before we merge?

*sigh* ...ok i guess

File: index.html [hunk 1]
```diff
@@ -17,7 +17,7 @@ <h1>BONEGANG Homepage</h1>
 
          <a href="assets/imgs/sven.jpeg">Contact information</a>
 
-         <img src="assets/imgs/sven.jpeg">
+         <img src="assets/imgs/sven.jpeg" alt="Swedish meatballs">
```
Comment (nefultre) [inline index.html:20]: lgtm!!!

**Pull request merged**